examples/test-LwIP-netconn/scpi_server.h

```diff
@@ -28,14 +28,29 @@
 
 #ifndef _SCPI_SERVER_H_
 #define _SCPI_SERVER_H_
+#ifdef __cplusplus
+extern "C" {
+#endif
 
+#define SCPI_DEVICE_PORT  5025 // scpi-raw standard port
+#define SCPI_CONTROL_PORT 5026 // libscpi control port (not part of the standard)
 
 #include <stdint.h>
+#include "api.h"
```

Comment:
This file `api.h` is not part of the PR

Reply:
This probably needs to be
```
#include "lwip/api.h"
```
for netconn to be defined.

Reply:
So, please, use `#include "lwip/api.h"` instead of bare `#include "api.h"`.